src/ovirtapi/transform.js

```diff
@@ -374,16 +374,15 @@ const CdRom = {
   toInternal ({ cdrom }: { cdrom: ApiCdRomType }): CdRomType {
     return {
       id: cdrom.id,
-      file: {
-        id: cdrom.file ? cdrom.file.id : '',
-      },
+      fileId: cdrom && cdrom.file && cdrom.file.id,
```

Comment:
If cdrom can be `null` or `undefined`, then, please, add same check to `id` and set type as `{ cdrom?: ApiCdRomType }`

Reply:
Updated the check assuming `cdrom` is passed (as per the flow type definition)